fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/NormalizeAggregate.java

```diff
@@ -279,8 +285,13 @@ private LogicalPlan normalizeAgg(LogicalAggregate<Plan> aggregate, Optional<Logi
         List<NamedExpression> upperProjects = normalizeOutput(aggregateOutput,
                 groupByExprContext, argsOfAggFuncNeedPushDownContext, normalizedAggFuncsToSlotContext);
 
-        // create a parent project node
-        LogicalProject<Plan> project = new LogicalProject<>(upperProjects, newAggregate);
+        ExpressionRewriteContext rewriteContext = new ExpressionRewriteContext(ctx);
+        LogicalProject<Plan> project = eliminateGroupByConstant(groupByExprContext, rewriteContext,
+                normalizedGroupExprs, normalizedAggOutput, bottomProjects, aggregate, upperProjects);
```

Comment:
current code is very weird, because we assemble plan in two different place. the better impl of is that we always assemble plan in it or out of it.